arbitrum-docs/build-decentralized-apps/how-tos/migrate-nfts-arbitrum.mdx

```diff
@@ -0,0 +1,137 @@
+---
+
+
+import PublicPreviewBannerPartial from '../../partials/_public-preview-banner-partial.md'; 
+
+<PublicPreviewBannerPartial />
+
+This how-to is for **developers** who want to migrate existing <a data-quicklook-from="non-fungible-tokens-nfts">NFT</a> collections to <a data-quicklook-from="arbitrum-one">Arbitrum One</a> or <a data-quicklook-from="arbitrum-nova">Nova</a>.
+
+This guidance applies to NFT collections used within web3 games, decentralized apps (dApps), and other use-cases.
+
+### Prerequisites
+
+ - An NFT collection that's minted and managed by smart contracts on either an **EVM-compatible** chain (like Ethereum mainnet) or a supported **non-EVM** chain[^1].
+ - NFTs that are implemented using the ERC-721 or ERC-1155 token standards[^2]. (todo - clarify how this prereq can be satisfied for non-EVM chains)
+
+
+:::info
+
+ERC-721 and ERC-1155 tokens **cannot be migrated to Arbitrum via [bridge.arbitrum.io](https://bridge.arbitrum.io/)**. The Arbitrum Bridge only supports ERC-20 tokens.
+
+For guidance on how to migrate ERC-20 tokens to Arbitrum, see [How to bridge ERC-20 tokens to Arbitrum](#todo) instead.
+
+:::
+
+
+## Why should I migrate my project's NFTs to Arbitrum?
+
+<!-- todo: this can be moved to a dedicated concept doc in the future -->
+
+There are three main reasons to migrate your your project's NFTs to Arbitrum:
+
+ 1. **Low fees**. NFTs hosted by one of Arbitrum's chains can be minted, transfered, and traded at a much lower cost compared to Ethereum mainnet.
+ 2. **Fast transactions**. From the end-user's perspective, transactions on Arbitrum's chains are processed more quickly than they are on Ethereum mainnet. This lets you build NFT-powered experiences that feel just about as fast as traditional apps and games.
+ 3. **Interoperability with the Ethereum and Arbitrum ecosystems**. If you're migrating your project's NFTs from a non-EVM chain, migration to Arbitrum will let your users freely roam between Ethereum dApps, carrying their NFTs along with them. More precisely, this means that your users' NFTs will be usable across dApp boundaries, as long as those dApps' smart contracts are hosted by EVM-compatible chains.
+
+<!-- todo: when shouldn't you? Risks to call out? -->
+
+
+## How do I migrate my project's NFTs to Arbitrum?
+
+There are a number of options to choose from, each with its own tradeoffs:
+
+1. **Set up cross-chain messaging between an old NFT contract to a new one on Arbitrum.** (todo - summary of tradeoffs, requirements, constraints - eg high engineering cost; seamless and unobtrusive UX for end-users; doesn't work if you're coming from a non-EVM chain, etc)
+2. **Mint a new collection on Arbitrum and distribute NFTs to existing holders.** (todo)
+3. **Build contracts on Arbitrum that depend on NFT ownership from other EVM chains.** (todo)
+4. **Building your own NFT Bridge.** (todo)
```

Comment:
I haven't seen Daniel's video yet, but isn't option 4 similar to option 1? In the sense that having your own bridge would require setting up cross-chain messaging between both blockchains.
Or maybe there's a difference that I'm missing!